<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HetUpdClass</class>
 <widget class="QDialog" name="HetUpdClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>661</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Input File</string>
   </property>
   <widget class="QLineEdit" name="infile">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input File Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selInfile">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>661</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Output File</string>
   </property>
   <widget class="QLineEdit" name="outdir">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="outfile">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="selOutdir">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selOutfile">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Directory</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output File Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>141</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Run Option</string>
   </property>
   <widget class="QRadioButton" name="compressRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Decompress</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="recompressRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Recompress</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rechunkRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Rechunk</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="verboseCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>verbose information</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>491</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>451</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Compression Options</string>
    </property>
    <widget class="QRadioButton" name="bzlibRadioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>321</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>use BZLIB compression</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="zlibRadioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>321</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>use ZLIB compression</string>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="chunkSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>4096</number>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="value">
     <number>65535</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>25</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Chunk Size</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="awsCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>481</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Strict AWSTAPE Specification (chunksize=4096,no compression)</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
